docs/rules/no-firestore-object-arrays.md

```
@@ -3,3 +3,64 @@
+```ts
+export type UserProfile = {
+  id: string;
+  tags: string[];
+  timestamps: Timestamp[];
+  friends: Record<string, { id: string; name: string; index: number }>;
+  contacts: { [id: string]: { email: string; index: number } };
+};
+```
+
+Invalid (arrays of objects):
+```ts
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)